build.assets/build-package.sh

```diff
@@ -61,10 +61,11 @@ LINUX_CONFIG_DIR=/etc
 LINUX_DATA_DIR=/var/lib/teleport
 
 # extra package information for linux
-MAINTAINER="ops@gravitational.com"
+MAINTAINER="info@gravitational.com"
 LICENSE="Apache License 2.0"
```

Comment:
Does this need to be `Apache-2.0` to show up in Ubuntu Package tool?

Reply:
Yes, I suspect it does.